### PR TITLE
refactor: scaffold one template

### PR DESCRIPTION
**Pull request by nliu-ms**
Scaffold only one template.

### CODE REVIEW

File: packages/fx-core/src/plugins/resource/appstudio/manifestTemplate.ts [hunk 1]
```diff
@@ -34,35 +34,57 @@ import {
...
+    let localAppName = appName;
+    if (suffix.length + appName.length <= TEAMS_APP_SHORT_NAME_MAX_LENGTH) {
+      localAppName = localAppName + suffix;
+    }
+    localManifestString = replaceConfigValue(localManifestString, "appName", localAppName);
+    const localManifest = JSON.parse(localManifestString);
+    await saveManifest(projectRoot, localManifest, true);
 
-  let remoteManifestString = TEAMS_APP_MANIFEST_TEMPLATE_V3;
-  remoteManifestString = replaceConfigValue(remoteManifestString, "appName", appName);
```
Comment (nliu-ms) [inline packages/fx-core/src/plugins/resource/appstudio/manifestTemplate.ts:63]: this step is not required, so just remove it.

File: packages/fx-core/src/plugins/resource/appstudio/manifestTemplate.ts [hunk 2]
```diff
@@ -34,35 +34,57 @@ import {
...
-    localAppName = localAppName + suffix;
-  }
-  localManifestString = replaceConfigValue(localManifestString, "appName", localAppName);
-  const localManifest = JSON.parse(localManifestString);
-  await saveManifest(projectRoot, localManifest, true);
+  if (isConfigUnifyEnabled()) {
+    const manifestString = TEAMS_APP_MANIFEST_TEMPLATE_V3;
+    const manifest = JSON.parse(manifestString);
+    if (existingApp) {
+      manifest.developer = {
```
Comment (HuihuiWu-Microsoft) [inline packages/fx-core/src/plugins/resource/appstudio/manifestTemplate.ts:65]: Is this all needed for existing app?

**Comment by nliu-ms** [comment]
E2E TEST: AzureAppScaffold

File: packages/fx-core/src/plugins/resource/appstudio/manifestTemplate.ts (hunk 2 above)
Comment (nliu-ms) [reply packages/fx-core/src/plugins/resource/appstudio/manifestTemplate.ts:65]: For existing app, it has different developer config:
image

Comment (HuihuiWu-Microsoft) [reply packages/fx-core/src/plugins/resource/appstudio/manifestTemplate.ts:65]: When will tab content updated to the template?

**Review by jayzhang — APPROVED** [review]

**Pull request merged**